sys/ps/ps.c

```diff
@@ -98,7 +98,8 @@ void ps(void)
             overall_used += stacksz;
 #endif
 #ifdef MODULE_SCHEDSTATISTICS
-            double runtime_ticks =  sched_pidlist[i].runtime_ticks / (double) xtimer_now() * 100;
+            double runtime_ticks = sched_pidlist[i].runtime_ticks /
```

Comment:
It seems like this only works for the first 2**32 ticks after boot, right? (Now that we're looking at it. :) )